apps/design/backend/README.md

```diff
@@ -3,3 +3,38 @@
 This backend is used by the [VxDesign frontend](../frontend) and isn't intended
 to be run on its own. The best way to develop on the backend is by running the
 frontend.
+
+## PostgreSQL Version
```

Comment:
Great instructions!

I know this may not be intuitive, but mind moving them to `apps/design/frontend`. In doing our README passes pre-cert, we decided to have the `apps/<app-name>/frontend` READMEs be the source of truth READMEs with dev tips and all, e.g., https://github.com/votingworks/vxsuite/tree/main/apps/central-scan/frontend.

And then every `apps/<app-name>/backend` README just points to the frontend, and includes nothing else.

And while you're at it, would you mind fixing this broken link in the VxDesign frontend README? I know it's been broken unrelated to this PR. Just remembered us talking about it a while back.

https://github.com/votingworks/vxsuite/blob/da20f07c856054ec18d01b540199e2aba2eb2aed/apps/design/frontend/README.md#L20